druid-shell/Cargo.toml

```diff
@@ -42,7 +42,6 @@ features = ["d2d1_1", "dwrite", "winbase", "libloaderapi", "errhandlingapi", "wi
 [target.'cfg(target_os="macos")'.dependencies]
 cocoa = "0.19.0"
 objc = "0.2.5"
-core-graphics = "0.18.0"
```

Comment:
Just out of curiosity, does the workspace feature somehow make this unnecessary / did you put this somewhere else?

Reply:
Good question, and no, this is an unrelated change I'm sneaking in.

I noticed while moving to the workspace that we were importing two versions of this, one directly and one via the `cocoa` crate. After a bit of digging it turned out that we weren't actually using this import, so I took the opportunity to delete.